docs/monitoring/rest_api.md

```diff
@@ -22,36 +22,69 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-Flink has a monitoring API that can be used to query status and statistics of running jobs, as well as recent completed jobs.
+Flink has a monitoring API that can be used to query the status and statistics of running jobs, as well as recent completed jobs.
 This monitoring API is used by Flink's own dashboard, but is designed to be used also by custom monitoring tools.
 
 The monitoring API is a REST-ful API that accepts HTTP GET requests and responds with JSON data.
 
 * This will be replaced by the TOC
 {:toc}
 
-
 ## Overview
 
-The monitoring API is backed by a web server that runs as part of the *JobManager*. By default, this server listens at post `8081`, which can be configured in `flink-conf.yaml` via `jobmanager.web.port`. Note that the monitoring API web server and the web dashboard web server are currently the same and thus run together at the same port. They respond to different HTTP URLs, though.
+<div class="codetabs" markdown="1">
+
+<div data-lang="JobManager" markdown="1">
+The jobManager monitoring API allows you to query the status and statistics of running jobs, as well as recent completed jobs.
+
+By default, this server binds to `localhost`` and listens at post `8081`, which can be configured in `flink-conf.yaml` via `jobmanager.web.address` and `jobmanager.web.port`.
 
 In the case of multiple JobManagers (for high availability), each JobManager will run its own instance of the monitoring API, which offers information about completed and running job while that JobManager was elected the cluster leader.
+</div>
+
+<div data-lang="HistoryServer" markdown="1">
```

Comment:
I think it's OK to keep it here as more detailed documentation, but I would vote to add a custom page which lists the relevant configuration options on a single page.